TypeScript/README.md

```diff
@@ -5,13 +5,114 @@ I heard that no one ignores TypeScript users. However, it was enough for me to b
 
 ### \<List>
 
+- [Web Speech API Practice (2024.11.14)](#web-speech-api-practice-20241114)
 - [`helloWorld("console.log")` (2024.05.23)](#helloworldconsolelog-20240523)
 - [Import a JSON file (2024.01.30)](#import-a-json-file-20240130)
 - [Touch Event Practice (2024.01.16)](#touch-event-practice-20240116)
 - [Big Block Lettering in Console (2023.05.28)](#big-block-lettering-in-console-20230528)
 - [Hello World (2023.02.28)](#hello-world-20230228)
 
 
+## [Web Speech API Practice (2024.11.14)](#list)
+
+- **Text-to-Speech(TTS)** conversion of Korean language input utilizing *Web Speech API*
+
+  ![Web Speech API Practice](./Images/WebSpeechAPIPractice.PNG)
+
+- Reference
+  - [MDN Web Docs](https://developer.mozilla.org/en-US/docs/) > [Web Speech API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Speech_API)
```

Comment:
- Add a sentence
  : As a feature natively supported by most browsers, it requires no additional installation or importing, making it usable even in offline environments such as airplane mode